go/coordinator/Makefile

```diff
@@ -0,0 +1,56 @@
+.PHONY: build
+build:
+	go build -v -o bin/chroma ./cmd
+
+test: build
+	go test -cover -race ./...
+
+lint:
+	#brew install golangci-lint
+	golangci-lint run
+
+clean:
+	rm -f bin/chroma
+
+docker:
+	docker build -t chroma-coordinator:latest .
+
+docker_multi_arch:
+	docker buildx build --platform linux/x86_64,linux/arm64 -t oxia:latest .
+
+.PHONY: proto
+proto:
```

Comment:
Should we should colocate the proto commands so that if the protos are changed in one language, the code is regenerated in downstream languages? I could go either way on this but jw